spec/support/xml_attribute_block_matcher.rb

```diff
@@ -111,6 +111,10 @@ module XMLBlockMatchers
     failure_message do
       SuperDiff::Differs::Hash.call(expected, actual_hash, indent_level: 0)
     end
+
+    failure_message_when_negated do
```

Comment:
Why have this method that does the same as the above `failure_message` method?

Reply:
just outputs the diff for devs to analyse in either scenario *(`.not_to` or `.to`)